### ISSUE TITLE
Can't acccess to user.css

### ISSUE THREAD

**Issue by qwertygc** [shaarli/Shaarli#993]
Hi!
We can't view personnal modifications. Indeed, the data directory is protected and we have a 403 error. Consequently, user.css isn't displayed and we have not personnal modifications :/

I have put this code in .htaccess and it's work!

```
Order allow,deny
Deny from all
<FilesMatch user\.css>
        Allow from all
</FilesMatch>
```

**Comment by virtualtam** [comment]
Thanks @qwertygc!

This partially addresses #872, I'll submit a PR to update the corresponding `.htaccess` for Apache HTTPD 2.2 & 2.4

@nodiscc @ArthurHoaro do you think this is worth fixing for Docker containers as well?

**Comment by virtualtam** [comment]
#996 submitted to fix Apache HTTPD 2.2 & 2.4 configuration

TODO:
- [ ] update Docker images' configuration
- [ ] update example Nginx configuration

**Comment by virtualtam** [comment]
Closing as a duplicate
